src/averageAges.js

```diff
@@ -14,12 +14,27 @@
  *
  * @return {number}
  */
+
+function getAverageAge(humans) {
+  const agesMap = humans.map(person => person.died - person.born);
+  const averageAges = agesMap.reduce(
+    (prev, next) => prev + next,
+    0) / agesMap.length;
+
+  return averageAges;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const toFilterMan = (person) => {
+    const isMan = person.sex === 'm';
+    const isCentury = Math.ceil(person.died / 100) === century;
+
+    return century ? isMan && isCentury : isMan;
```

Comment:
1 - avoid writing ternary operators in one line, it's harder to read
2 - `isMan` duplicated, move it out of ternary